src/sentry/runner/initializer.py

```diff
@@ -384,6 +386,20 @@ def initialize_app(config, skip_service_validation=False):
     env.data["start_date"] = timezone.now()
 
 
+def configure_arroyo():
+    # Arroyo is configured in such a central place because
+    #
+    # 1) it doesn't harm any process that doesn't use arroyo
+    # 2) we want arroyo to be fully configured in subprocesses of the multiprocessing consumer
+    from arroyo import configure_metrics
+
+    from sentry.sentry_metrics.metrics_wrapper import MetricsWrapper
+    from sentry.utils.metrics import backend
+
+    metrics_wrapper = MetricsWrapper(backend, "sentry_metrics.indexer")
```

Comment:
so I know we don't have arroyo being used for any other consumers in sentry, but if that does happen, is this going to be a problem?

Reply:
I don't think so. If there is no consumer running, this doesn't do anything. If some other team starts using arroyo, metrics will work out of the box.